code/go/0chain.net/miner/protocol_block_main.go

```diff
@@ -274,22 +289,22 @@ func (mc *Chain) generateBlock(ctx context.Context, b *block.Block,
 	if config.DevConfiguration.IsFeeEnabled {
 		err = mc.processTxn(ctx, mc.createFeeTxn(b), b, blockState, iterInfo.clients)
 		if err != nil {
-			return err
+			logging.Logger.Error("generate block (payFees)", zap.Int64("round", b.Round), zap.Error(err))
```

Comment:
yep I thought about it, bit it is much better, than network timeout due to absent proposal, so I decided that network liveness is high priority here,

Reply:
okay, we can fire an issue to document this behavior in case some day we want to re-consider about this or have better solution for the network liveness.

Reply:
Hmm, I'm still worried about the failure of reward and fee transactions here, we can ignore failures of the transaction for commiting the config updates, but ignoring the reward transaction errors may need further check. @guruhubb  what do you think.

Reply:
I think this is fine @peterlimg.  If there is less time to generate the block, the miner needs to discard time-consuming transactions and just add their reward transaction and send it otherwise they won't receive their reward.  

There is no reason for a miner to discard a lot of transactions as they will earn less fees.

Reply:
@guruhubb it's not about we don't have not enough time to generate the block I think. The generator has already finished most of the work for packing a block, the last thing it tries to do is create and execute the reward/fee transaction so that miners would get rewarded. The question here is whether we should allow this transaction to fail during the block generation process. 

If yes, then generators and miners will create and verify this block for free, no miners would get rewarded. If no, then this block generator will failed to pack the block, and other generators will create the block successfully with all miners get rewarded. Anyway, I prefer to do `no`.

Reply:
Ok got it. Yes miners won't be happy if reward/fee transaction is not executed, but if it is a code execution issue due to MPT growth, then the whole chain will stall. I think if it fails, then the miner can take less time for the transactions (say half of what it started out) for the next block it generates. @peterlimg

Reply:
@guruhubb yes, take less time for the transactions in next block generation might be an option, we can do this later. @dabasov what do you think.

Reply:
yep, I agree to do it later